### PR TITLE
fix #198496: add db piccolo flute

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -64,6 +64,22 @@
                   <genre>jazz</genre>
                   <genre>orchestra</genre>
             </Instrument>
+                  <Instrument id="db-piccolo">
+                        <longName>D♭ Piccolo</longName>
+                        <shortName>D♭ Picc.</shortName>
+                        <description>Piccolo Flute in Db</description>
+                        <musicXMLid>wind.flutes.flute.piccolo.dflat</musicXMLid>
```
Comment (lasconic) [inline share/instruments/instruments.xml]: this xml id doesn't exist. https://www.musicxml.com/for-developers/standard-sounds/

Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Ok, will fix this after the weekend by dropping the `dflat`

Comment (lasconic) [inline share/instruments/instruments.xml]: Thanks. If you could squash too, that would be perfect!

Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: done

**Pull request merged**